### ISSUE TITLE
Rebase on zowe CLI lts-incremental

### ISSUE THREAD

**Issue by matthewpwilson** [IBM/zowe-cli-cics-deploy-plugin#121]
Allow our plugin to work with the lts-incremental branch of Zowe CLI. This means customers will be able to use the supported build as suggested by the Zowe docs.

This will mean we'll need to change our dependencies from `@zowe` back to `@brightside`

Note: we should wait until our Zowe CLI PRs get merged into lts-incremental first.

**Comment by matthewpwilson** [comment]
I need https://github.com/zowe/zowe-cli-cics-plugin/pull/29 to be merged before we can move our @zowe/cics dependency to @brightside/cics lts-incremental version

**Comment by ChrisPark89** [comment]
To update: It's merged now.
